web-framework-commons/src/main/webapp/js/alfresco.js

```diff
@@ -8359,9 +8359,9 @@ Alfresco.util.Ajax = function()
          }
 
          if (serverResponse.status === 0 &&
-             document.referrer.length > 0 &&
-             Alfresco.constants.AIMS_AUTH_SERVER_URL !== undefined &&
-             document.referrer.startsWith(Alfresco.constants.AIMS_AUTH_SERVER_URL)
+             serverResponse.statusText === "communication failure" &&
```

Comment:
is it possible "communication failure" may be localized? Do we have the possibility do check by status code instead?

Reply:
@alexbalmus Good point. Tested with setting browser on another language, but the "communication failure" status message that comes from YUI connector manager is not localized. However I removed the statusText condition since it is redundant, status related condition is enough.